templates/layouts/default.pug

```diff
@@ -91,6 +91,7 @@ html
                 ul.dropdown-menu.nav-account-dropdown-menu.pull-right(role="menu")
                   if username
                     li: span.welcome-text #{username}
+                    li: a(href="/account/report") Report form
```

Comment:
Don't call it `Report form`. Nobody will know what that is.
Maybe `Report user`.